test/common/testlib.py

```diff
@@ -93,6 +93,21 @@
 opts.jobs = 1
 opts.fetch = True
 
+default_layouts = [
+    {
+        "name": "desktop",
+        "is_mobile": False,
+        "shell_size": [1920, 1200],
+        "content_size": [1680, 1130]
```

Comment:
Again, these two don't seem independent from each other -- the delta should be pretty much fixed, and depend on our shell/menu CSS. If we can't determine these automatically (e.g. by waiting for overview to render and then querying the content size), let's at least add a comment how to compute these.

Reply:
These sizes are a "contract" with the reference images.  We guarantee that the content iframe is this exact size whenever assert_pixels takes a screenshot, and thus the reference image for a full frame is this exact size.  We can not dynamically compute the content size, but we do dynamically compute the window size that is necessary to get this exact content size.

The content size here has been determined by looking at the current sizes of reference images for the whole content iframe.

Reply:
Ah, I see -- so shell tests would resize to shell_size, and page tests to content_size? That makes sense. (Let's document it, but we already have a conversation about it, so resolving)